spec/features/student_performs_an_exam_spec.rb

```diff
@@ -1,31 +1,49 @@
 require "spec_helper"
 
 feature "student performs an exam" do
-  given!(:ruby)  { Fabricate(:course, title: "Introduction to Ruby") }
-  given!(:rails) { Fabricate(:course, title: "Rapid Prototyping") }
-  given!(:week1) { Fabricate(:quiz, course: ruby, title: "Week 1-Procedural") }
-  given!(:week2) { Fabricate(:quiz, course: ruby, title: "Week 2-OOP") }
-  given!(:q1) { Fabricate(:question, quiz: week1, points: 3, content: "1+1") }
-  given!(:q2) { Fabricate(:question, quiz: week1, points: 4, content: "2+2") }
-  given!(:q3) { Fabricate(:question, quiz: week1, points: 5, content: "3+3") }
-
-  given!(:a1) do
-    Fabricate(:answer, question: q1, correct: true, content: "answer is 2")
-  end
-  given!(:wrong_answer) do
-    Fabricate(:answer, question: q2, correct: false, content: "w")
-  end
-  given!(:a2) do
-    Fabricate(:answer, question: q2, correct: true, content: "answer is 4")
-  end
-  given!(:a3) do
-    Fabricate(:answer, question: q3, correct: true, content: "answer is 6")
+  given(:ruby)  { Fabricate(:course, title: "Introduction to Ruby") }
+  given(:rails) { Fabricate(:course, title: "Rapid Prototyping") }
+  given(:week1) do
+    Fabricate(:quiz, course: ruby, title: "Week 1-Procedural") do
+      questions {
+        [
+          Fabricate(:question, points: 3, content: "1+1") do
+            answers {[
+              Fabricate(:answer, correct: true, content: "answer is 2"),
+              Fabricate(:answer, correct: false, content: "x"),
+              Fabricate(:answer, correct: false, content: "x"),
+              Fabricate(:answer, correct: false, content: "x"),
+              Fabricate(:answer, correct: false, content: "x")
+            ]}
+          end,
+          Fabricate(:question, points: 4, content: "2+2") do
+            answers {[
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.